packages/reactive-element/src/reactive-element.ts

```diff
@@ -668,6 +668,12 @@ export abstract class ReactiveElement
     (this.__controllers ??= []).push(controller);
   }
 
+  removeController(controller: ReactiveController) {
+    if (this.__controllers !== undefined) {
+      this.__controllers = this.__controllers?.filter((c) => c !== controller);
+    }
```

Comment:
This kind of makes me want to switch back to a Set 🤔

Reply:
Iterating the array is slightly faster. Let's leave it for now.